web/src/buddy_data.ts

```diff
@@ -142,6 +142,15 @@ function get_num_unread(user_id: number): number {
 }
 
 export function user_last_seen_time_status(user_id: number): string {
+    // Check if the user is deactivated
+    if (!people.is_person_active(user_id)) {
+        // Determine if user is a bot or human
+        const person = people.get_by_user_id(user_id);
+        if (person.is_bot) {
+            return $t({defaultMessage: "This bot has been deactivated."});
+        }
+        return $t({defaultMessage: "This user has been deactivated."});
+    }
```

Comment:
The function `user_last_seen_time_status` sounds like a function supposed to return us the last seen status of a given user. So, let's avoid returning statements such as "This user has been deactivated." from this function. We can do this directly in the `get_title_data` function or as an alternative, pass the `is_bot` and `is_deactivated` params to the `web/templates/buddy_list_tooltip_content.hbs` and handle the content there.

Reply:
Hmm, makes sense. I think doing this directly in `get_title_data` would be a better option.